amd/src/mutations.js

```diff
@@ -247,6 +280,9 @@ export default class {
                 cardid: cardId,
                 timestamp: timestamp,
             },
+            fail: () => {
+                this.processUpdateFail();
```

Comment:
I think you are missing to pass the stateManager here? Same a few lines below?

Reply:
Fixed now, thank you!